solr/CHANGES.txt

```diff
@@ -176,6 +176,10 @@ Bug Fixes
 * SOLR-17629: If SQLHandler failed to open the underlying stream (e.g. Solr returns an error; could be user/syntax problem),
   it needs to close the stream to cleanup resources but wasn't. (David Smiley)
 
+* SOLR-17519: SolrJ fix to CloudSolrClient and HTTP ClusterState where it can fail to request cluster state
+  if its current live nodes list is stale. HTTP ClusterState now retains the initial configured list of passed URLs as backup
```

Comment:
I don't think Solr users know about "HTTP ClusterState"; it's not likely in their vocabulary.  On the other hand, wording like "SolrJ CloudSolrClient configured with Solr URLs" is likely to be understood.